packages/playground/src/utils/date.ts

```diff
@@ -0,0 +1,10 @@
+export default function dateFiler(value: number): string {
+  const formater = Intl.DateTimeFormat("en", {
+    timeStyle: "short",
+    dateStyle: "short",
+    // eslint-disable-next-line
+  } as any);
+
+  // eslint-disable-next-line
+  return formater.format(new Date(value * 1000));
+}
```

Comment:
Please use moment.js to and combine both methods of date in a single file
```suggestion
export function toHumanDate(timeInSeconds: number): string {
  return moment(timeInSeconds * 1000).format("M/D/YY, h:m A")
}
```